python/google/protobuf/pyext/extension_dict.h

```diff
@@ -47,7 +47,8 @@ namespace protobuf {
 class Message;
 class FieldDescriptor;
 
-using internal::shared_ptr;
+using std::shared_ptr;
```

Comment:
It was complaining about internal::shared_ptr being not found. I am open to suggestions on how to fix this.